lib/widgets/content.dart

```diff
@@ -797,7 +807,7 @@ class _InlineContentBuilder {
       // and our parser doesn't.  So don't do anything here.
       return const TextSpan(text: "");
     } else if (node is StrongNode) {
-      return _buildStrong(node);
+      return _buildStrong(ambientTextStyle: widget.style, node);
```

Comment:
No need to pass this to one of this class's own methods — it can look at `widget.style` directly, just like this `_buildNode` method can.